templates/web-secrets.yaml

```diff
@@ -15,6 +15,16 @@ data:
   session-signing-key: {{ .Values.secrets.sessionSigningKey | b64enc | quote }}
   worker-key-pub: {{ .Values.secrets.workerKeyPub | b64enc | quote }}
 
```

Comment:
nit: we want to keep unneeded whitespace to a minimum ([source](https://helm.sh/docs/chart_template_guide/control_structures/#controlling-whitespace))
```suggestion
```